packages/webpack-plugin/src/plugin.ts

```diff
@@ -294,23 +294,20 @@ export class StylableWebpackPlugin {
         const options =
             loadStylableConfig(compiler.context, (config) => {
                 return isWebpackConfigProcessor(config)
-                    ? config.webpackPlugin(defaults, compiler)
+                    ? config.webpackPlugin(defaults, compiler, getTopLevelInputFilesystem(compiler))
                     : undefined;
             })?.config || defaults;
 
         this.options = options;
     }
-    private createStcBuilder(compiler: Compiler) {
-        if (!this.options.stcConfig) {
-            return;
-        }
-
+    private getStylableConfig(compiler: Compiler) {
         const configuration = resolveStcConfig(
             compiler.context,
-            typeof this.options.stcConfig === 'string' ? this.options.stcConfig : undefined
+            typeof this.options.stcConfig === 'string' ? this.options.stcConfig : undefined,
+            getTopLevelInputFilesystem(compiler)
         );
 
-        if (!configuration) {
+        if (this.options.stcConfig && !configuration) {
```

Comment:
same as rollup - not sure this ever gets here

Reply:
removing as well.